app/models/date_aggregation.rb

```diff
@@ -21,8 +21,8 @@ def history(days_to_display = 7)
     stats
   end
 
-  def increment(amount = 1)
-    redis.multi { keys.each { |key| redis.incrby key, amount }}
+  def increment(amount = 1, date: Time.zone.today)
```

Comment:
Should we always require the `date` argument to avoid these kind of issues in the future?